handlers/auth.ts

```diff
@@ -1,19 +1,19 @@
 import { Handler } from "openapi-backend";
+// TODO: Figure out a better way to handle the type checking of the OpenAPI
 import type * as T from "../types/openapi.js";
 import {
   getApi,
-  getCurrentBlockNumber,
   getNetwork,
   getNonce,
   getProviderHttp,
   getProviderKey,
+  getSiwfUrl,
 } from "../services/frequency.js";
-import { generateChallenge, createAuthToken, getMsaByPublicKey, useChallenge } from "../services/auth.js";
+import { createAuthToken } from "../services/auth.js";
 import { AnnouncementType } from "../services/dsnp.js";
 import { getSchemaId } from "../services/announce.js";
 import { getIpfsGateway } from "../services/ipfs.js";
-import { signatureVerify } from "@polkadot/util-crypto";
-import { hexToU8a, numberToU8a } from "@polkadot/util";
+import { validateSignup, validateSignin } from "@amplica-labs/siwf";
```

Comment:
weird I do not see validateSignup in the siwf.

Reply:
It is in https://github.com/AmplicaLabs/siwf/pull/160